io/net/scp_ssh.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Prudhvi Miryala<mprudhvi@linux.vnet.ibm.com>
+#
+
+# check the ssh into peer machine
+# check the scp into peer machine
+
+import time
+import hashlib
+from avocado import main
+from avocado import Test
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils import process
+
+
+class ScpTest(Test):
+    '''
+    check the ssh into peer
+    check the scp into peer
+    '''
+    def setUp(self):
+        '''
+        To check and install dependencies for the test
+        '''
+        sm = SoftwareManager()
+        for pkg in ["openssh-clients", "net-tools"]:
```

Comment:
distro's may have different package name is it handled ? ignore this comment if "openssh-clients", "net-tools" package names are common in all distributions ?